apps/previewgenerator.sh

```diff
@@ -16,67 +16,218 @@ debug_mode
 # Check if root
 root_check
 
-msg_box "Please note that this will put your server at risk as it will install a package called 'imagick' which is known to have several flaws.
+msg_box "This script will install the previewgerator. 
 
-You can check this issue to understand why: https://github.com/nextcloud/vm/issues/743
-
-You can choose to cancel installing this in the next step."
-if [[ "no" == $(ask_yes_or_no "Do you still want to continue?") ]]
+It can speedup the feel of Nextcloud by a lot."
+if [[ "no" == $(ask_yes_or_no "So do you want to install the previewgenerator?") ]]
 then
     exit
 else
-    sleep 1
+    # Install preview generator and ffmpeg
+    install_and_enable_app previewgenerator
+    occ_command config:system:set enable_previews --value=true --type=boolean
+    install_if_not ffmpeg
 fi
 
-# Install preview generator
-install_and_enable_app previewgenerator
+msg_box "In the next step you can choose to install a package called imagick to speed up the generation of previews and get support for more filetypes. 
 
-# Run the first preview generation and add crontab
-if [ -d "$NC_APPS_PATH/previewgenerator" ]
```

Comment:
Why was this check removed?

Reply:
I copied this to line 200, because I think there is the only command for which you really need the previewgenerator enabled.
If you want, I can reverse this change.

Reply:
I moved it back to the start of the script.
So: Done.